docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,149 @@
+---
+title:  Merchant onboarding
+description: This document describes the onboarding process for merchants and gives step-by-step instructions about completing it.
+template: concept-topic-template
+last_updated: Mar 29, 2023
+---
+
+This document describes how to onboard merchants.
```

Comment:
change to: This document describes the process for the marketplace operator how to onboard merchants.